README.md

```diff
@@ -99,41 +136,71 @@ into the element. This is the only method that must be implemented by subclasses
 ## API Documentation
 
 See the [source](https://github.com/PolymerLabs/lit-element/blob/master/src/lit-element.ts#L90)
- for detailed API info, here are some highlights. Note, the leading underscore
- is used to indicate that these methods are
- [protected](https://en.wikipedia.org/wiki/Class_(computer_programming)#Member_accessibility);
- they are not private and can and should be implemented by subclasses.
- These methods generally are called as part of the rendering lifecycle and should
- not be called in user code unless otherwise indicated.
-
-  * `_createRoot()`: Implement to customize where the
+ for detailed API info, here are some highlights.
+
+  * `createRenderRoot()` (protected): Implement to customize where the
   element's template is rendered by returning an element into which to
   render. By default this creates a shadowRoot for the element.
   To render into the element's childNodes, return `this`.
 
-  * `_firstRendered()`: Called after the element DOM is rendered for the first time.
-
-  * `_shouldRender(props, changedProps, prevProps)`: Implement to control if rendering
-  should occur when property values change or `invalidate` is called.
+  * `shouldUpdate(changedProperties)` (protected): Implement to control if updating and rendering
+  should occur when property values change or `invalidate` is called. The `changedProps`
```

Comment:
changedProps => changedProperties (you missed that during rename)

Reply:
Done